frontend/src/lib/components/SortableList/sortableListItem.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+
+import { useElementBoundingRect } from "@lib/hooks/useElementBoundingRect";
+import { createPortal } from "@lib/utils/createPortal";
+import { resolveClassNames } from "@lib/utils/resolveClassNames";
+import { DragIndicator } from "@mui/icons-material";
+
+import { HoveredArea, SortableListContext } from "./sortableList";
+import { SortableListDropIndicator } from "./sortableListDropIndicator";
+
+export type SortableListItemProps = {
+    id: string;
+    title: React.ReactNode;
+    initiallyExpanded?: boolean;
+    startAdornment?: React.ReactNode;
+    endAdornment?: React.ReactNode;
+    children: React.ReactNode;
+};
+
+/**
+ *
+ * @param {SortableListItemProps} props Object of properties for the SortableListItem component (see below for details).
+ * @param {string} props.id ID that is unique among all components inside the sortable list.
+ * @param {React.ReactNode} props.title Title component of the list item.
+ * @param {React.ReactNode} props.startAdornment Start adornment to display to the left of the title.
+ * @param {React.ReactNode} props.endAdornment End adornment to display to the right of the title.
+ * @param {React.ReactNode} props.children Child components to display as the content of the list item.
+ *
+ * @returns {React.ReactNode} A sortable list item component.
+ */
+export function SortableListItem(props: SortableListItemProps): React.ReactNode {
+    const [isExpanded, setIsExpanded] = React.useState<boolean>(props.initiallyExpanded ?? true);
+
+    const divRef = React.useRef<HTMLDivElement>(null);
+    const boundingClientRect = useElementBoundingRect(divRef);
+
+    const sortableListContext = React.useContext(SortableListContext);
+
+    const isHovered = sortableListContext.hoveredElementId === props.id;
+    const isDragging = sortableListContext.draggedElementId === props.id;
+    const dragPosition = sortableListContext.dragPosition;
+
+    function handleToggleExpanded() {
+        setIsExpanded(!isExpanded);
+    }
```

Comment:
This is never called, since the generic toggle-adornment was removed from `<Header>`. Should either make "expanded" be a prop, leaving it fully up to the implementer; or make it possible for adornments to use this callback (and probably having a default toggle-adornment if so)